packages/launchpad/src/setup-actions.ts

```diff
@@ -131,7 +131,7 @@ export const setUpActionsForTasks = ( {
 							apiVersion: '1.1',
 							method: 'post',
 						} );
-						window.location.assign( `/home/${ siteSlug }?celebrateLaunch=true` );
+						setCelebrateLaunchModalIsOpen?.( true );
```

Comment:
The naming here feels too tightly-coupled to this specific implementation/code path. I think we should call this something more generic like `siteLaunched?.()`, as the implementation is simply notifying the caller that the site was launched. For the pre-launch launchpad context, we can then translate that into calls that relate to the celebration modal.

The "wrapper" here could be as simple as something like the following in the caller:
```
const siteLaunched = () => {
  setCelebrateLaunchModalIsOpenWrapper( true );
};
```